community/views.py

```diff
@@ -8,21 +8,104 @@
     get_org_name,
     get_owner,
     get_upstream_deploy_url,
+    get_remote_url
 )
+from data.models import Team, Contributor
+
+
+def initialize_org_context_details():
+    org_name = get_org_name()
+    org_details = {
+        'name': org_name,
+        'blog_url': 'https://blog.{}.io/'.format(org_name),
+        'twitter_url': 'https://twitter.com/{}_io/'.format(org_name),
+        'facebook_url': 'https://www.facebook.com/{}Analyzer'.format(
+            org_name),
+        'repo_url': get_remote_url().href,
+        'docs': 'http://{}.io/docs'.format(org_name),
+        'newcomer_docs': 'http://{}.io/newcomer'.format(org_name),
+        'coc': 'http://{}.io/coc'.format(org_name),
+        'logo_url': ('https://api.{}.io/en/latest/_static/images/'
+                     '{}_logo.svg'.format(org_name, org_name)),
+        'gitter_chat': 'https://gitter.im/{}/{}/'.format(org_name,
+                                                         org_name),
+        'github_core_repo': 'https://github.com/{}/{}/'.format(org_name,
+                                                               org_name),
+        'licence_type': 'GNU AGPL v3.0'
+    }
+    return org_details
+
+
+def get_header_and_footer(context):
+    context['isTravis'] = Travis.TRAVIS
+    context['travisLink'] = Travis.TRAVIS_BUILD_WEB_URL
+    context['org'] = initialize_org_context_details()
+    print('Running on Travis: {}, build link: {}'.format(context['isTravis'],
+                                                         context['travisLink']
+                                                         ))
+    return context
 
 
 class HomePageView(TemplateView):
     template_name = 'index.html'
 
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        context['isTravis'] = Travis.TRAVIS
-        context['travisLink'] = Travis.TRAVIS_BUILD_WEB_URL
+    def get_team_details(self):
```

Comment:
Add a `related_name` in the teams field of the `Contributor` model.

Get a `Team` by it's `name`.

Get `contributors` of that `team` by using the `related_name`.